demo/index.html

```diff
@@ -308,10 +353,10 @@ <h2>Error handling</h2>
         </ul>
         <div class="tab-content mt-2">
           <div class="tab-pane example-07-result show active">
-            <div class="row align-items-center">
+            <div class="row align-items-start">
               <print-spec
                 class="col-8"
-                editable="false"
+                expanded="false"
                 select="Error layers"
```

Comment:
Noticed that renaming the object ids broke the select.

```suggestion
                select="Spec with invalid sources"
```